app/server/lib/filenamesUtils.ts

```diff
@@ -0,0 +1,7 @@
+function removesNonASCII(filename: string){
+    return filename.replace(/[^\x20-\x7E]/g, '').replace('%', '');
```

Comment:
`replace('%', '')` only replaces one `%`. Also `%` is an ascii character so the function name is misleading.